### DISCUSSION TITLE
Site reloading continuously

### DISCUSSION THREAD

**Discussion by ARH-MNAJS** [sanidhyy/duolingo-clone#7] [Q&A] [answered]
My site is being continuously reloading after I run npm run dev

**Comment by sanidhyy** [comment]
Try opening in the incognito tab.

**Reply by ARH-MNAJS** [reply]
I did still same issue. Upon debugging got the issue in Clerk as the auth state is stuck in interstitial state. the reason is cross origin refferrer 

The official logs show
Options debug, {
    "debug": true,
    "beforeAuth": false,
    "afterAuth": false
  }
  authenticateRequest state is interstitial, {
    "status": "interstitial",
    "reason": "cross-origin-referrer",
    "message": "",

**Reply by sanidhyy** [reply]
Does it says something like this in debug message - `URL debug`?

**Reply by ARH-MNAJS** [reply]
no

**Reply by ARH-MNAJS** [reply]
am using project idx is that the issue?

**Reply by sanidhyy** [reply]
Yes your Origin url is not matching with referrer url

**Reply by sanidhyy** [reply, marked as answer]
If you wanna use web based editor, I used Gitpod to create it and that worked for me. You can try it too.

https://gitpod.io
